tests/testutil/service.go

```diff
@@ -302,6 +397,24 @@ func (ts *TestService) GetKubeCurrentWeight() int {
 	return getRevisionWeight(app, ts.Version)
 }
 
+// WaitForScaleDown waits until either 5 minutes pass or a service has scaled down to its minimum value
```

Comment:
Is 5 minutes because of the scale down to 0?

Reply:
Yeah it usually takes approx. 3 minutes to go from 1 pod to 0, but I gave it some room in case this is used on a service that started with more pods.

Reply:
Would it be easier to just send this info into `WaitForScale` ?

Reply:
I tried that originally. `WaitForScale` doesn't quite do what we want though because it returns the `Spec.Scale` value, which is still 1 even if the service scales down to 0.